src/foam/box/RemoteException.js

```diff
@@ -30,3 +30,39 @@ foam.CLASS({
     }
   ]
 });
+
+// TODO: Once modelled exceptions are merged this can be removed in
+//       favour of an exception modelled in foam.nanos.cron
+foam.CLASS({
+  package: 'foam.box',
+  name: 'CapabilityRequiredRemoteException',
+  implements: ['foam.core.Exception'],
+  properties: [
+    {
+      class: 'String',
+      name: 'id',
```

Comment:
Yes, but wow I totally forgot about that; this field should be renamed.

Reply:
Oh, actually it needs to be called `id` because that's what SessionClientBox expects. It's really just the name of the exception, meant to match `id` property from `RemoteException`

Reply:
Is there a reason this doesn't extend `RemoteException`? It seems to me that it should, especially considering it has `RemoteException` in the name.